workflows/VGP-assembly-v2/Pre-Curation_PretextMap/PretextMap_Generation-tests.yml

```diff
@@ -0,0 +1,84 @@
+- doc: Test outline for PretextMap-Generation-from-2-haplotypes
+  job:
+    Haplotype 1:
+      class: File
+      location: https://zenodo.org/records/14218240/files/Haplotype%201.fasta
+      filetype: fasta
+    Haplotype 2:
+      class: File
+      location: https://zenodo.org/records/14218240/files/Haplotype%202.fasta
+      filetype: fasta
+    HiC reads - forward:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: HiC forward reads
+        location: https://zenodo.org/records/14218240/files/Hi-C%20forward%20reads.fastqsanger.gz
+    HiC reads - reverse:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: HI-C reverse reads
+        location: https://zenodo.org/records/14218240/files/HI-C%20reverse%20reads.fastqsanger.gz
```

Comment:
Why is this not a paired collection ?

Reply:
Because we are aligning the two separately as single reads to the reference. If we use a paired collection BWA-MEM2 will align them as paired reads with an expected distance between the pairs and that won't work for Hi-C

Reply:
Are you basing this pipeline on something existing ? Otherwise `-SP` would disable pairing and mate rescue.

Reply:
Yes, it's based on the pipeline they have been using for the VGP. It does the same separate alignment in scaffolding workflow, and it has a specific tool for merging alignments for Arima Hi-C reads.  Also the workflow 4, assembly with HiC, needs them to be in 2 separate collections, so it may be easier for the user to use the same collections throughout the whole process. We can discuss using the -SP option for a future release, we'd need to test is works the same.

Reply:
Yeah, I would bring that up. What tool is this ?

Reply:
Filter and merge https://usegalaxy.org/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/bellerophon/bellerophon/1.0+galaxy0